### ISSUE TITLE
Feature Request:  Set price for adding Autoload

### ISSUE THREAD

**Issue by ADDgamer45** [loki79uk/FS25_UniversalAutoload#34]
If it is at all possible can you add a configurable purchase price to add autoload to a trailer? 

Like a yes or no to add autoload then a price config in the mod settings?

I want to allow autoload in my multi-player server but I want there to be a cost to it.

**Comment by loki79uk** [comment]
Yes, that is definitely one of my planned features.  You can set a price for loading each pallet already if you want to, but you have to do do it via xml (the same as in 22).  I will make everything configurable in the store with different options like bake trailer, etc. And could certainly add a price for them as well.

**Comment by clavicus-zz** [comment]
is there already a line in the xml to enable this? i can't find any xml files in the mod folder either... probably just my inexperience..

**Comment by ddewar** [comment]
> is there already a line in the xml to enable this? i can't find any xml files in the mod folder either... probably just my inexperience..

Check in your farming simulator directory above the mods folder. There is a modSettings folder that should have the universal autoload xml file.

**Comment by DuffCal** [comment]
This would be great if I could set it per item loaded.  I looked in the XML and didn't see anything there.  Would you be able to give me any guidance on where I could find this setting?

https://gist.github.com/DuffCal/bccb07ad22b8c7ad5c8986e1e221eb27

**Comment by loki79uk** [comment]
I think setting a price for adding autoload is just a bit too abstract to add as a feature.  If you like some kind of realism, then paying per item is much closer to paying workers in real life, and it scales perfectly with how much you use it and the trailer capacity, etc.  Paying "$500,000" as a target after which you start to make more profit seems a bit arbitrary to me.
